lib/ProductOpener/Config.pm

```diff
@@ -32,6 +32,10 @@ if (not defined $flavor) {
 	die("The PRODUCT_OPENER_FLAVOR_SHORT environment variable must be set.");
 }
 
+# %options will be redefined by autoload below
+# but we need it to avoid Perl complaining
+%ProductOpener::Config:: % options = ();
```

Comment:
@stephanegigandet our commits did cross, and so I realize this wasn't needed !

Reply:
@alexgarel ah right, here is a PR to remove it: https://github.com/openfoodfacts/openfoodfacts-server/pull/11345